shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitParameters.java

```diff
@@ -48,9 +58,22 @@ public static ShortCircuitParameters load(PlatformConfig platformConfig) {
 
         ModuleConfig config = platformConfig.getOptionalModuleConfig("short-circuit-parameters").orElse(null);
         if (config != null) {
+            //TODO: add parameter for a list of equipments on which the analysis should be done (sytematic study but with specified equipments)
             parameters.setSubTransStudy(config.getBooleanProperty("subTransStudy", ShortCircuitConstants.SUBTRANS_STUDY));
             parameters.setWithFeederResult(config.getBooleanProperty("withFeederResult", ShortCircuitConstants.WITH_FEEDER_RESULT));
-
+            parameters.setStudyType(config.getEnumProperty("study-type", ShortCircuitConstants.StudyType.class, DEFAULT_STUDY_TYPE));
+            if (parameters.studyType == ShortCircuitConstants.StudyType.SELECTIVE_STUDY) {
```

Comment:
I have a very high level interrogation : 

selective and systematic studies are quite different things, from the feature point of view (comparable to loadflow and security analysis).

Maybe it would be a better idea to separate them in 2 different APIs, with possibly different inputs and result structures ?

Reply:
Another, also high level consideration:

I think all those parameters which define the fault would find a better place as an argument of the computation (like contingencies), than in parameters.

Parameters should only be pretty generic parameters of the computation tool, not something that will often change from one computation to another.

Reply:
@sylvlecl : this is slightly different from a loadflow because in a loadflow, results of systematic analysis are obtained by re-running a new LF computation for each fault. Here, one single computation (one matrix inversion) is valid for all faults of the same type (ex: triphased). Additional info results from a post processing of mandatory information. 
But from a user point of view, I don't know what is the best...

I agree with @sylvlecl , we need to seperate parameters that define the tool general behaviour and input data that describes each contingency.

Parameters:
- subTransStudy
- Type of Norm used (IEC, IEEE, RTE =COURCIRC)
- shunt/no shunt
- type of voltage profile for short circuit computation (provided in input or use of nominal values)

General Input data for shortcircuit analysis
- SELECTIVE versus SYSTEMATIC
- equipment if SELECTIVE
- "mandatory result data only" or "detailed data" if selective (could be set to "detailed data" by default if selective)
- fault impedance (resistance + reactance)
- fault location
- Type of shortcircuit: triphased, monophased, biphased, biphased ground. Later, additional information will be needed to fully describe unbalanced shortcircuits.

Reply:
In case of a selective short circuit analysis, we will have a dedicated `run` method, that we have to design. I agree, we can dedicate a new object to fault description, that will be in the argument of the run: fault location, fault impedance. Do you think we have to add the type of short-circuit calculation here too? Triphased, monophased, biphased, biphased ground. Or in parameters ?

Reply:
In the same run, you could ask to simulate a fault for triphased, biphased etc. and take the most conservative result. It is maybe better to link it to a fault rather then a parameter of the model. In the long term, it would be nice to have systematic shortcircuit analysis mixing different types of balanced and unbalanced faults